common/dependabot-common.gemspec

```diff
@@ -18,8 +18,8 @@ Gem::Specification.new do |spec|
   spec.require_path = "lib"
   spec.files        = []
 
-  spec.required_ruby_version = ">= 2.7.0"
-  spec.required_rubygems_version = ">= 2.7.3"
+  spec.required_ruby_version = ">= 3.1.0"
```

Comment:
Do we need it to be the _minimum_ version? It'll require all consumers of the gem to also update, and if our changes are backwards compatible (which they seem to be?) we wouldn't need to force the upgrade on them

Reply:
I'm not sure if we _need_ it to be bumped, I am simply obeying the `Rubocop`: https://github.com/dependabot/dependabot-core/pull/5447#discussion_r936824160

We could disable that specific cop, but given that other folks can keep using the previous release of Dependabot, I think it's okay (🤞) if we bump this. That way if we introduce backwards breaking changes later (such as [the new shorthand syntax](https://github.com/dependabot/dependabot-core/pull/5447#discussion_r935202378)), then we don't start silently breaking code for others.

That said, IIUC, currently our internal service that consumes this lib isn't on 3.1 yet, so holding back this part of the change allows us to merge this rather than having to first update the internal consumer... but again IMO, I'd kinda rather just update the internal service since it should be fairly straightforward... from what I could tell, the `2.7`->`3.1` changes were mostly straightforward despite the major version bump.

Reply:
Yes, definitely. I think either bumping both or bumping none is fine. But bumping the RuboCop config only has the risk @jeffwidman mentioned.